src/uiprotect/api.py

```
@@ -1402,6 +1402,11 @@ async def update(self) -> Bootstrap:
             self.__dict__.pop("bootstrap", None)
             self._bootstrap = bootstrap
 
+            try:
+                await self.poll_events()
+            except Exception as err:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)